SQLite/Com.weboniselab.hemant.android.SQLiteAssignment/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <string name="app_name">SQLiteAssignment</string>
+    <string name="hello_world">Hello world!</string>
+    <string name="action_settings">Settings</string>
+    <string name="title_activity_details_from_sqlite">DetailsFromSQLite</string>
+    <string name="Title">Details Stored in Data Base</string>
+    <string name="toast_string_dbstrated">Data Saving started....</string>
+    <string name="toast_reading">Reading all Details..</string>
+    <string name="da_string">[Id: %d, Name: %s, Age %d, Height  %d, Weight %d]\n</string>
```

Comment:
Have look at http://stackoverflow.com/questions/3656371/dynamic-string-using-string-xml

This will explain you how to format the string.